test/dashboard-menu.test.js

```diff
@@ -0,0 +1,35 @@
+import { html, fixture, expect } from '@open-wc/testing';
+import Sinon, { stub } from 'sinon';
+import { Router } from '@vaadin/router';
+import '../src/dashboard/Dashboard-menu.js';
+
+describe('Dashboard Menu details', () => {
+  // Write test cases inside this block
+  // refer basic-details.js files
+  let el;
+  before( async() => {
+    el = await fixture(html `<dashboard-menu><dashboard-menu>`);
+  });
+  
+  it('is accessible', () => {
+    expect(el).to.be.accessible;
+  });
+
+  it('_setTypeInLS() method called ', () => {
+    const spy = Sinon.spy(el, '_setTypeInLS');
```

Comment:
Just call el._setTypeInLS() and expect navigateToDetails() is called. Only navigateToDetails() need to be spied. This `const spy = Sinon.spy(el, '_setTypeInLS');` not needed.